app/components/candidate_interface/complete_section_component.html.erb

```diff
@@ -1,8 +1,17 @@
-<%= form_with model: @application_form, url: @path, method: @request_method do |f| %>
-  <div class='govuk-form-group'>
-    <%= f.hidden_field @field_name, value: false %>
-    <%= f.govuk_check_box @field_name, true, multiple: false, label: { text: t(checkbox_label) } %>
-  </div>
+<%= form_with model: section_complete_form, url: path, method: request_method do |f| %>
+  <%= f.govuk_error_summary %>
+
+  <% if summary_component %>
+    <%= render(summary_component) %>
+  <% end %>
```

Comment:
If the summary_component is optional, would it make sense to not make it a dependency of the CompleteSectionComponent (and instead render them both separately in the view template as we do currently)?

Related question: where are we rendering this without providing a summary_component?

Reply:
Hey @malcolmbaig - the summary_component should not be optional - I'll fix